apps/workspaces/apis/export_settings/serializers.py

```diff
@@ -38,6 +38,7 @@ class ExpenseGroupSettingsSerializer(serializers.ModelSerializer):
     reimbursable_export_date_type = serializers.CharField(allow_null=True, allow_blank=True, required=False)
     corporate_credit_card_expense_group_fields = serializers.ListField(allow_null=True, required=False)
     ccc_export_date_type = serializers.CharField(allow_null=True, allow_blank=True, required=False)
+    ccc_expense_state = serializers.CharField(allow_null=True, allow_blank=True, required=False)
```

Comment:
Add the same line for `expense_state` as well . 
```suggestion
    expense_state = serializers.CharField(allow_null=True, allow_blank=True, required=False)
```